lib/vrl/stdlib/src/uuid_v4.rs

```diff
@@ -38,28 +38,25 @@ impl Expression for UuidV4Fn {
     }
 }
 
-// #[cfg(test)]
-// mod tests {
-//     use super::*;
-//     use crate::map;
-//     use std::convert::TryFrom;
-
-//     vrl::test_type_def![static_def {
```

Comment:
Do we not want these type def tests anymore?

Reply:
Ah, yes. Normally not because the `test_function` now has a `tdef` field that tests the expected type def. However this function couldn't use the normal `test_function` since we don't know the exact result until we run it.

I'll put the type_def test back in.